modules/reference/pages/console/config.adoc

```diff
@@ -3,10 +3,10 @@
 :page-aliases: console:reference/config.adoc
 :page-categories: Redpanda Console
 
-The `redpanda-console-config.yaml` file contains configuration properties for Redpanda Console. 
+The `redpanda-console-config.yaml` file contains configuration properties for Redpanda Console.
 
-- If you install Redpanda Console on Linux, this file is installed on each instance in `/etc/redpanda/redpanda-console-config.yaml`. 
-- If you install Redpanda Console in a containerized environment, you must mount this file to the `/etc/redpanda/` directory in the container.
+- If you install Redpanda Console on Linux, this file is installed on each instance in `/etc/redpanda/redpanda-console-config.yaml`.
+- If you install Redpanda Console in a containerized environment, you must mount this file to the `/etc/redpanda/` directory in the container. You must also provide the filepath to your configuration file by setting either the `-config.filepath` flag or the `CONFIG_FILEPATH` environment variable.
```

Comment:
```suggestion
- If you install Redpanda Console in a containerized environment, you can mount this file to the `/etc/redpanda/` directory in the container. You must also provide the filepath to your configuration file by setting either the `-config.filepath` flag or the `CONFIG_FILEPATH` environment variable.
```

NIT: Technically it can be anywhere where Console can read from.

If you want to be more precise, it's also possible to specify configs via env variables only. This is why Console would still start if you don't specify a CONFIG filepath as it assumes that it should only source the config from environment variables.